Update to nmcli > 0.5.0 causes test failures

Tried updating nmcli in requirements.txt to v1.1.2 but get following errors in unit tests:

```
==================================== ERRORS ====================================
_ ERROR collecting tests/gatewayconfig/bluetooth/test_wifi_services_characteristic.py _
tests/gatewayconfig/bluetooth/test_wifi_services_characteristic.py:11: in <module>
    class TestWifiServicesCharacteristic(TestCase):
tests/gatewayconfig/bluetooth/test_wifi_services_characteristic.py:20: in TestWifiServicesCharacteristic
    ???
E   TypeError: __init__() missing 2 required positional arguments: 'bssid' and 'freq'
_ ERROR collecting tests/gatewayconfig/bluetooth/test_wifi_ssid_characteristic.py _
tests/gatewayconfig/bluetooth/test_wifi_ssid_characteristic.py:10: in <module>
    class TestWifiSSIDCharacteristic(TestCase):
tests/gatewayconfig/bluetooth/test_wifi_ssid_characteristic.py:19: in TestWifiSSIDCharacteristic
    ???
E   TypeError: __init__() missing 2 required positional arguments: 'bssid' and 'freq'
```

This seems to be caused by an updated listed in the [changelog](https://pypi.org/project/nmcli/1.1.2/#:~:text=lang%3A%20str)%20%2D%3E%20None-,Change%20Log,-1.1.2) under version 0.6.0 as `Added BSSID and FREQ to DeviceWifi.`

Can we update the test cases to handle this?

--- a/requirements.txt
+++ b/requirements.txt
@@ -5,7 +5,7 @@ dbus-python==1.3.2
 gpiozero==1.6.2
 h3==3.7.6
 hm-pyhelper==0.13.45
-nmcli==0.5.0
+nmcli==1.1.2
 protobuf==3.20.3
 pycairo==1.23.0
 requests==2.28.1

--- a/tests/gatewayconfig/bluetooth/test_wifi_services_characteristic.py
+++ b/tests/gatewayconfig/bluetooth/test_wifi_services_characteristic.py
@@ -17,10 +17,10 @@ class TestWifiServicesCharacteristic(TestCase):
     @patch('lib.nmcli_custom.device.wifi_rescan')
     @patch('lib.nmcli_custom.device.wifi',
            return_value=[
-               DeviceWifi(in_use=False, ssid='TP-LINK', mode='Infra', chan=5, rate=54, signal=100,
-                          security='WPA1'),
-               DeviceWifi(in_use=True, ssid='PHICOMM', mode='Infra', chan=1, rate=44, signal=96,
-                          security='WPA2')
+               DeviceWifi(in_use=False, ssid='TP-LINK', bssid='001122334455', mode='Infra',
+                          chan=5, freq=2432, rate=54, signal=100, security='WPA1'),
+               DeviceWifi(in_use=True, ssid='PHICOMM', bssid='001122334456', mode='Infra',
+                          chan=1, freq=2412, rate=44, signal=96, security='WPA2')
            ])
     def test_ReadValue(self, mock1, mock2):
         characteristic = WifiServicesCharacteristic(self.service, GatewayconfigSharedState())

--- a/tests/gatewayconfig/bluetooth/test_wifi_ssid_characteristic.py
+++ b/tests/gatewayconfig/bluetooth/test_wifi_ssid_characteristic.py
@@ -16,10 +16,10 @@ class TestWifiSSIDCharacteristic(TestCase):
     @patch('lib.nmcli_custom.device.wifi_rescan')
     @patch('lib.nmcli_custom.device.wifi',
            return_value=[
-               DeviceWifi(in_use=False, ssid='TP-LINK', mode='Infra', chan=5, rate=54, signal=100,
-                          security='WPA1'),
-               DeviceWifi(in_use=True, ssid='PHICOMM', mode='Infra', chan=1, rate=44, signal=96,
-                          security='WPA2')
+               DeviceWifi(in_use=False, ssid='TP-LINK', bssid='001122334455', mode='Infra',
+                          chan=5, freq=2432, rate=54, signal=100, security='WPA1'),
+               DeviceWifi(in_use=True, ssid='PHICOMM', bssid='001122334456', mode='Infra',
+                          chan=1, freq=2412, rate=44, signal=96, security='WPA2')
            ])
     def test_ReadValue(self, mock1, mock2):
         characteristic = WifiSSIDCharacteristic(self.service, GatewayconfigSharedState())

